code/game/objects/structures/vehicles/secway.dm

```diff
@@ -0,0 +1,63 @@
+/obj/structure/bed/chair/vehicle/secway
+	name = "secway"
+	desc = "A brave security cyborg gave its life to help you look like a complete tool."
+	icon_state = "secway"
+	keytype = /obj/item/key/security
+	var/clumsy_check = 1
+
+/obj/item/key/security
+	name = "secway key"
+	desc = "A keyring with a small steel key, and a rubber stun baton accessory."
+	icon_state = "keysec"
+
+
+/obj/structure/bed/chair/vehicle/secway/update_mob()
+	if(!occupant)
+		return
+
+	switch(dir)
+		if(SOUTH)
+			occupant.pixel_x = 0
+			occupant.pixel_y = 3
+		if(WEST)
+			occupant.pixel_x = 2
+			occupant.pixel_y = 3
+		if(NORTH)
+			occupant.pixel_x = 0
+			occupant.pixel_y = 3
+		if(EAST)
+			occupant.pixel_x = -2
+			occupant.pixel_y = 3
+
+
+/obj/structure/bed/chair/vehicle/secway/handle_layer()
+	if(dir == WEST || dir == EAST || dir == SOUTH)
+		layer = FLY_LAYER
+		plane = PLANE_EFFECTS
+	else
+		layer = OBJ_LAYER
+		plane = PLANE_OBJ
+
+
+/obj/structure/bed/chair/vehicle/secway/Bump(var/atom/obstacle)
+	..()
+	
+	if(istype(obstacle, /mob/living))
```

Comment:
You realize that this will cause it to stun any mob it touches, whether or not the occupant is clumsy/drunk, right?

Not saying that's a bad thing, necessarily, just wanted to make sure that it's intentional.